charts/kube-otel-stack/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kube-otel-stack
 description: Chart for sending Kubernetes metrics to Lightstep using the OpenTelemetry Operator.
 type: application
-version: 0.2.13
-appVersion: 0.74.0
+version: 0.2.15
+appVersion: 0.73.0
```

Comment:
Moving this back to `0.73.0` since we're not updating the collector versions. `main` also has a later version, so this change will get overridden anyways.